orchagent/natorch.cpp

```diff
@@ -3269,6 +3269,70 @@ void NatOrch::queryHitBits(void)
     }
 }
 
+void NatOrch::updateAllConntrackEntries(void)
+{
+    SWSS_LOG_ENTER();
+
+    /* Send notifcations for the Single NAT entries to set timeout */
+    NatEntry::iterator natIter = m_natEntries.begin();
+    while (natIter != m_natEntries.end())
+    {
+
+        if ((natIter->second.nat_type == "snat") and (natIter->second.addedToHw == true) and
+            (natIter->second.entry_type != "static"))
+        {
+            SWSS_LOG_ERROR("Update %s NAT entry [ip %s]", natIter->second.nat_type.c_str(), natIter->first.to_string().c_str());
+            std::vector<FieldValueTuple> fvVector;
+            std::string key = natIter->first.to_string();
+            setTimeoutNotifier->send("SET-SINGLE-NAT", key, fvVector);
```

Comment:
It looks like the Timeout value is set to max for active entries in queryhitBits(), why do we need to update the timeout here again

Reply:
The timeout is set to max value (432000) when entries got added in NatMgr code, in queryhitBits() we are deleting the entries which are not active. Here in this section of code (funtion updateAllConntrackEntries() ) called for every 86400 secs (1 day) to set all dynamic active entries conntrack timeout to Max value (432000).